iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorModelHolder.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.iidm.network.ShuntCompensatorModel;
+import com.powsybl.iidm.network.ShuntCompensatorModelType;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+interface ShuntCompensatorModelHolder extends ShuntCompensatorModel {
+
+    ShuntCompensatorModelType getType();
+
+    ShuntCompensatorModel getModel();
+
+    <M extends ShuntCompensatorModel> M getModel(Class<M> modelType);
+
+    void setShuntCompensator(ShuntCompensatorImpl shuntCompensator);
```

Comment:
Not sure this method should be available

Reply:
See comment above